build/deps/deps.MODULE.bazel

```diff
@@ -0,0 +1,81 @@
+# This file is included from the root MODULE.bazel
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "sqlite3",
+    build_file = "//:build/BUILD.sqlite3",
+    patch_args = ["-p1"],
+    patches = [
+        "//:patches/sqlite/0001-row-counts-plain.patch",
+        "//:patches/sqlite/0002-macOS-missing-PATH-fix.patch",
+        "//:patches/sqlite/0003-sqlite-complete-early-exit.patch",
+        "//:patches/sqlite/0004-invalid-wal-on-rollback-fix.patch",
+    ],
+    sha256 = "f59c349bedb470203586a6b6d10adb35f2afefa49f91e55a672a36a09a8fedf7",
+    strip_prefix = "sqlite-src-3470000",
+    url = "https://sqlite.org/2024/sqlite-src-3470000.zip",
+)
+
+
+http_archive(
+    name = "nbytes",
+    build_file = "//:build/BUILD.nbytes",
+    sha256 = "34be48071c86add2f8d14fd4a238c47230965fd743a51b8a1dd0b2f0210f0171",
+    strip_prefix = "nbytes-0.1.1",
+    url = "https://github.com/nodejs/nbytes/archive/refs/tags/v0.1.1.tar.gz",
+)
+
+http_archive(
+    name = "ncrypto",
+    sha256 = "b438cf71b1c24036e388f191a348cdc76aca75310eabca0fef5d81d5032a5d20",
+    strip_prefix = "ncrypto-1.0.1",
+    type = "tgz",
+    url = "https://github.com/nodejs/ncrypto/archive/refs/tags/1.0.1.tar.gz",
+)
+
+# ========================================================================================
+# tcmalloc
+http_archive(
+    name = "com_google_tcmalloc",
+    integrity = "sha256-29cSZUwbEyiW8Y7FneaAzNNYLHeBmAPqBuIciHeE/u0=",
+    # repo_mapping = {"@com_google_absl": "@abseil-cpp"},
+    strip_prefix = "google-tcmalloc-cf3dc2d",
+    type = "tgz",
+    url = "https://github.com/google/tcmalloc/tarball/cf3dc2d98bd64cb43f4f98db0acaf5028a7b81eb",
+)
+
+http_archive(
+    name = "dragonbox",
+    build_file_content = """cc_library(
+            name = "dragonbox",
+            hdrs = glob(["include/dragonbox/*.h"]),
+            visibility = ["//visibility:public"],
+            include_prefix = "third_party/dragonbox/src",
+        )""",
+    integrity = "sha256-LxBEjWZTVbQfWZ6GmseIA/gvE7Bwzn71rntczrihePM=",
+    strip_prefix = "dragonbox-6c7c925b571d54486b9ffae8d9d18a822801cbda",
+    url = "https://github.com/jk-jeon/dragonbox/archive/6c7c925b571d54486b9ffae8d9d18a822801cbda.zip",
```

Comment:
nit: we've been using .tar.gz files when fetching from GitHub, let's keep that consistent